### ISSUE TITLE
Re-perform AVT1 tests with DAP on components

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#1308]
We've performed these tests in the past but want to re-run them on our markup and document the test results just to be safe 😄 This will also help us define the requirements and markup needed for our spec workflow (hopefully!).

This epic will keep track of the AVT1 tests on our components. The basic process for our AVT1 tests will be the following:

- Decide on a component variation below to run, create an issue for it (should we make a template for AVT1 issues?)
- Link the issue to this epic, assign yourself, and mark as in progress in ZenHub
- Link the issue under the variant name for the component in the component status table
- Open up a suitable environment to run DAP in (TBD, should we use Vanilla or React?)
- Run the DAP extension (or the AI enabled one) using the latest ruleset (I think 2018 August)
- Download the report in JSON format and place in: https://ibm.ent.box.com/folder/56014617250
  - The naming scheme should be `component-name_variant.json`
- If the report is clear, then place a ✅ in the corresponding column in: https://ibm.box.com/s/mhpgjlgss354mcidsv6rvpuu0bxlmfm4
- If the report shows problems, then:
  - Create an issue detailing the a11y failures (feel free to include recommendations on fixes too)
  - Mark the column with an ❌ and afterwards provide a link to the GitHub Issue under the text `GitHub Issue`
- If the fix is quick, feel free to make an accompanying PR
- When done, feel free to close the issue and check off the component in the list below!

## Component variations

- [x] Accordion #1317 
- [x] Breadcrumb
- [x] Button
  - [x] Primary
  - [x] Secondary
  - [x] Tertiary
  - [x] Ghost
  - [x] Danger
  - [x] Primary Danger
  - [x] Small variants
- [x] Checkbox
- [x] Code Snippets
  - [x] Code Snippet
  - [x] Inline Code Snippet
    - [x] Handle inside text with a light background
  - [x] Multi-line Code Snippet
- [x] Content Switchers
  - [ ] Content Switcher
  - [ ] Content Switcher with Icon
- [x] Copy Button
- [x] Data Table
  - [ ] With row expansion
  - [ ] With pagination
  - [ ] With sorting
  - [ ] With selection
  - [ ] Condensed table
  - [ ] Toolbar
- [x] Date Pickers
  - [ ] Simple Date Picker
  - [ ] Single Date Picker
  - [ ] Range Date Picker
  - [ ] Time Picker
- [x] Dropdowns
  - [ ] Dropdown
  - [ ] Inline Dropdown
  - [ ] Multi-select
  - [ ] Inline Multi-select
  - [ ] Filter Dropdown
- [x] File Uploader
- [x] Link
- [x] List
- [x] Loading
  - [ ] Loader
  - [ ] Inline Loader
  - [ ] Skeleton states
- [x] Modals
  - [ ] Passive Modal
  - [ ] Transactional Modal
  - [ ] Danger Modal
- [x] Notifications
  - [ ] Inline Notification
  - [ ] Toast Notification
- [x] Number Input
- [x] Overflow Menu
- [x] Pagination
- [x] Progress Indicator
- [x] Radio Button
- [x] Search
  - [ ] Small search
  - [ ] Large search
- [x] Select
  - [ ] Select
  - [ ] Inline select
- [x] Slider
- [x] Structured List
  - [ ] Structured list
  - [ ] Structured list with selection
- [x] Tabs
- [x] Tags
- [x] Text Area
- [x] Text Input
- [x] Tile
  - [ ] Clickable
  - [ ] Selectable
  - [ ] Expandable
- [x] Toggle
  - [ ] Toggle
  - [ ] Small toggle
- [x] Tooltip
  - [ ] Tooltip
  - [ ] Icon tooltip
  - [ ] Definition tooltip

**Comment by joshblack** [comment]
From our planning session, each task will have an estimate of `3`!

**Comment by joshblack** [comment]
🎉
